lib/vcr/util/internet_connection.rb

```diff
@@ -1,31 +1,25 @@
 module VCR
   # Ruby 1.8 provides Ping.pingecho, but it was removed in 1.9.
```

Comment:
why keeping this comment ?

Reply:
I think it's useful to provide background. With only the other comment (`This is copied, verbatim, from Ruby 1.8.7's ping.rb.`), it may not immediately obvious why this strategy is applied.